ansible/postdeploy.yml

```diff
@@ -6,4 +6,8 @@
 - hosts: ansible
   tasks:
     - include: tasks/installCatalog.yml
-      when: mode == "deploy"
\ No newline at end of file
+      when: mode == "deploy"
+      
```

Comment:
The ultimate goal is that the catalog installation scripts are removed out of openwhisk. However, the current phase is a transition state. We need to keep both of them tentatively, otherwise if we miss either of them, we cannot make sure all the actions are installed.

For the actions installed from the openwhisk-catalog, their test cases will be verified in the travis ci of the openwhisk-catalog, not in the openwhisk. The travis CI in openwhisk-catalog is responsible to guarantee the quality of the actions installed from the catalog.